### ISSUE TITLE
Add cmdline option to create a new dehydrated config directory

### ISSUE THREAD

**Issue by darix** [dehydrated-io/dehydrated#837]
at least until we get multi CA support

`dehydrated --init-config /path/to/dir/`

create all the subdirectories with all the proper permissions. copy a skeleton config from `/usr/share/dehydrated/config/`

**Comment by lukas2511** [comment]
Mh... not entirely sure how to actually implement this.

If there would be a defined path for all of the example files (like the `/usr/share` path you suggested) this would be easy, but there are tons of setups out there where this isn't the case and the script often is installed standalone without the rest of the repository...

For a good solution I'd need to somehow package the example files inside the main script. I'm working on splitting dehydrated into multiple source files, making it easier to work on it, with a script bundling everything back together into one big script. As part of that "build" script I could add the example files with escaped newlines to the script... mh...

**Comment by darix** [comment]
tbh if you split out files anyway i would use /usr/share/dehydrated/ for code and /usr/share/dehydrated/skeleton/ for the base config and not merge the files into one big script anymore.

**Comment by lukas2511** [comment]
No, dehydrated will still be delivered as a single shell-script, but it will be constructed from multiple parts just to make it easier to work on. Super simple build system, maybe even just a `cat * > ../dehydrated` :D
